MDS_3/include/CGAL/MDS_3/tet_soup_to_c3t3.h

```diff
@@ -188,8 +188,7 @@ bool build_finite_cells(Tr& tr,
           ++k;
         } while(f[0] != n0);
 
-        typename boost::unordered_map<std::array<int,3>, Surface_patch_index>::const_iterator
-          it = border_facets.find(f);
+        typename FacetPatchMap::const_iterator it = border_facets.find(f);
```

Comment:
Nowadays, I am pretty lazy with iterator types, and I use always `auto`:

```suggestion
        auto it = border_facets.find(f);
```

No need to actually change your code. That is just a remark.